tools/nats-bench/nats-bench.go

```diff
@@ -80,15 +99,27 @@ func main() {
 	// Run Subscribers first
 	startwg.Add(*numSubs)
 	for i := 0; i < *numSubs; i++ {
-		go runSubscriber(&startwg, &donewg, opts, *numMsgs, *msgSize)
+		nc, err := nats.Connect(*urls, opts...)
+		if err != nil {
+			log.Fatalf("Can't connect: %v\n", err)
+		}
+		defer nc.Close()
+
+		go runSubscriber(nc, &startwg, &donewg, *numMsgs, *msgSize)
 	}
 	startwg.Wait()
 
 	// Now Publishers
 	startwg.Add(*numPubs)
 	pubCounts := bench.MsgsPerClient(*numMsgs, *numPubs)
 	for i := 0; i < *numPubs; i++ {
-		go runPublisher(&startwg, &donewg, opts, pubCounts[i], *msgSize)
```

Comment:
Why did you remove helpers here?

Reply:
go-nats-example/nats-bench.go was out of sync with go-nats version, the current version is a copy from go-nats. Helper is still there